package-lock.json

```diff
@@ -2327,17 +2327,15 @@
       "dependencies": {
         "@oclif/config": {
           "version": "1.13.0",
-          "resolved": "https://registry.npmjs.org/@oclif/config/-/config-1.13.0.tgz",
```

Comment:
Should probably revert the changes here, I guess its caused by using a diff npm version

Reply:
Good call, thanks!